SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    
+    var newConversationButton: UIButton!
+    var feedbackLabel: UILabel!
+    var searchController: UISearchController! // TODO: For later
+    var feedbackTableView: UITableView!
+        
+    var feedbackData: [Feedback]!
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    var countEditTaps: Int = 0
+    var isFeedback: Bool!
+    let cellHeight: CGFloat = 75
```

Comment:
for a constant like this (only used once in the code) I'd prefer we just directly use 75 wherever it's being called. In this case it's in the `heightForRowAt` function. @kevinchan159 Thoughts?